[4.0] Fix site title

lufog opened the pull request:
Fix site title
before
![before](https://user-images.githubusercontent.com/10379487/29991859-ac2d0a70-8f97-11e7-9b70-6f5cfb494ae0.PNG)
after
![after](https://user-images.githubusercontent.com/10379487/29991862-be73ddf8-8f97-11e7-9662-4caf7a5af4c3.PNG)

lufog committed:
Update template.css

lufog committed:
Update template.css

lufog committed:
Update template.min.css

lufog committed:
Update _header.scss

lufog committed:
Update _header.scss

lufog committed:
Update template.css

lufog committed:
Update template.min.css

ghost commented:
changed Title meaningful.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17831">issues.joomla.org/tracker/joomla-cms/17831</a>.</sub>

ciar4n commented on templates/aurora/css/template.css:
```diff
@@ -8199,9 +8199,10 @@ legend {
-    font-size: 2rem;
-    color: #fefefe; }
+    margin-right: 3rem; }
+    .hgeader .navbar-brand a {
```
`.hgeader`? I'm guessing this should be `.header`?

ciar4n commented:
@DebuggeR94 You appear to have some differences between the SCSS and CSS. Have you edited the SCSS and then compiled to CSS or simply edited both manually?

C-Lodder commented:
Please do not try and manually edit the`template.css`. It will get automatically generated when you run:
`grunt compile:site`

lufog committed:
Update template.css

lufog commented:
@ciar4n Yes, I fixed the files straight through the Web interface, and made a typo. In the future, I will do as it should be :)

C-Lodder commented:
@DebuggeR94 - Please fix the error reported by Hound too

wilsonge committed:
Fix hound

lufog committed:
Fix language filter

Otherwise, the links looks like:
   site.domain/profile/en?view=profile
Instead of:
   site.domain/en/profile?view=profile

lufog commented:
Sorry, I'm still not quite clear with the github interface, and instead of create new pull request I sent another fix here.
Unfortunately I do not know how to remove it.

wojsmol commented:
@DebuggeR94 Just do new commit and revert https://github.com/joomla/joomla-cms/pull/17831/commits/9cee0d1c9251266d4c517a8086fee2803768081d.

lufog committed:
revert changes

Quy commented:
I assume this is no longer applicable.

Quy commented:
@franz-wohlkoenig Close since this is for the no longer available aurora template?

ghost commented:
closed as stated above.

thanks for Hint @Quy.
